src/server/debug/server/server.go

```diff
@@ -82,7 +90,14 @@ func (s *debugServer) handleRedirect(
 			if filter != nil {
 				switch f := filter.Filter.(type) {
 				case *debug.Filter_Pachd:
-					return collectPachd(ctx, tw, pachClient, pachdContainerPrefix)
+					if err := collectPachd(ctx, tw, pachClient, pachdContainerPrefix); err != nil {
+						return err
+					}
+					if wantDatabase {
+						if err := s.collectDatabaseStats(ctx, tw, databasePrefix); err != nil {
+							return err
+						}
+					}
```

Comment:
I think we want to return s. collectDatabaseStats here even if it's not an error right?